x-pack/plugins/security_solution/public/cases/components/case_view/index.test.tsx

```diff
@@ -111,8 +146,8 @@ describe('CaseView ', () => {
     usePostPushToServiceMock.mockImplementation(() => ({ isLoading: false, postPushToService }));
     useConnectorsMock.mockImplementation(() => ({ connectors: connectorsMock, isLoading: false }));
     useQueryAlertsMock.mockImplementation(() => ({
-      isLoading: false,
-      alerts: { hits: { hists: [] } },
+      loading: false,
+      data: { hits: { hits: [...alertsHit] } },
```

Comment:
nit: no need to spread here
```suggestion
      data: { hits: { hits: alertsHit } },
```